composer.json

```diff
@@ -85,9 +85,12 @@
         "ci:tests:coverage": "phpunit --do-not-cache-result --coverage-clover=coverage.xml",
         "ci:tests:sof": "phpunit --stop-on-failure --do-not-cache-result",
         "ci:tests:unit": "phpunit --do-not-cache-result",
+        "fix": [
+            "@fix:php"
+        ],
         "fix:php": [
-            "@fix:php:fixer",
-            "@fix:php:rector"
+            "@fix:php:rector",
+            "@fix:php:fixer"
```

Comment:
F comes before R in the latin alphabet, so the existing ordering was correct.

Reply:
I changed this on purpose:

> Also run Rector before the style fixer as Rector is responsible for structural changes that need to be style-fixed after that.

Reply:
> I changed this on purpose:
> 
> > Also run Rector before the style fixer as Rector is responsible for structural changes that need to be style-fixed after that.

Sorry, I missed that, and for some reason thought the script content itself would be reordered by Composer-Normalize :/